microsoftgraph/microsoft-graph-devx-api
# Merge branch Dev into Master

* update script to generate raptor report to login with managed ID
* Change used pool to 1-es hosted pool
* Other commits other than https://github.com/microsoftgraph/microsoft-graph-devx-api/commit/11a5cdc36e2e596378aadc0ae94c7ef61ec15018 describing changes above

### Overview
- Merge Commits from dev into master branch

## Testing Instructions
Among other changes, the below one is expected
* When [Snippet gen pipeline](https://dev.azure.com/microsoftgraph/Graph%20Developer%20Experiences/_build?definitionId=90&_a=summary) is run with base as master branch, the 3rd stage, 'GenerateCategorizationReport', should run successfully to completion

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoftgraph/microsoft-graph-devx-api/pull/2110)
